src/widgets/range-input/__tests__/range-input-test.ts

```diff
@@ -94,8 +99,9 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/range-input
       attribute,
     });
 
-    widget.init({ helper, instantSearchInstance });
-    widget.render({ results, helper });
+    widget.init!(createInitOptions({ helper, instantSearchInstance }));
+    // @ts-expect-error
```

Comment:
can you add which error is expected?

Reply:
Sure, added here: https://github.com/algolia/instantsearch.js/pull/4766/commits/bfaa2f34f463c5d56ada3f8fda37d7225010d10f